builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -151,6 +151,72 @@ local function get_setting_info(name)
 end
 
 
+-- These must not be translated, as they need to show in the local
+-- language no matter the user's current language.
+get_setting_info("language").option_labels = {
+	[""] = fgettext_ne("(Use system language)"),
+	--ar = " [ar]", blacklisted
+	be = "Беларуская [be]",
+	bg = "Български [bg]",
+	ca = "Català [ca]",
+	cs = "Česky [cs]",
+	cy = "Cymraeg [cy]",
+	da = "Dansk [da]",
+	de = "Deutsch [de]",
+	--dv = " [dv]", blacklisted
+	el = "Ελληνικά [el]",
+	en = "English [en]",
+	eo = "Esperanto [eo]",
+	es = "Español [es]",
+	et = "Eesti [et]",
+	eu = "Euskara [eu]",
+	fi = "Suomi [fi]",
+	fil = "Wikang Filipino [fil]",
+	fr = "Français [fr]",
+	gd = "Gàidhlig [gd]",
+	gl = "Galego [gl]",
+	--he = " [he]", blacklisted
+	--hi = "हिन्दी [hi]", blacklisted
+	hu = "Magyar [hu]",
+	id = "Bahasa Indonesia [id]",
+	it = "Italiano [it]",
+	ja = "日本語 [ja]",
+	jbo = "Lojban [jbo]",
+	kk = "Қазақша [kk]",
+	--kn = "ಕನ್ನಡ [kn]", blacklisted
+	ko = "한국어 [ko]",
+	ky = "Kırgızca / Кыргызча [ky]",
+	lt = "Lietuvių [lt]",
+	lv = "Latviešu [lv]",
+	mn = "Монгол [mn]",
+	mr = "मराठी [mr]",
+	ms = "Bahasa Melayu [ms]",
+	--ms_Arab = "Malay (Jawi) [ms_Arab]", blacklisted
```

Comment:
The difference is in the script.
```suggestion
	--ms_Arab = "Bahasa Melayu (Jawi) [ms_Arab]", blacklisted
```